app/auth/user_management.py

```
@@ -0,0 +1,80 @@
+    email: EmailStr
+    first_name: str
+    last_name: str
+
+
+class BulkUserRequest(BaseModel):
+    user_ids: List[str]
+    action: str
+
+
+@router.get("/profile")
+async def get_user_profile(current_user: Dict[str, Any] = Depends(get_current_user)):
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)